### PR TITLE
Push button routine, more cleanups

### PR DESCRIPTION
**Pull request by kyogithub**

### CODE REVIEW

File: LibreCO2_SCD30/LibreCO2_SCD30_ArduinoUNO.ino [hunk 1]
```diff
@@ -315,3 +289,35 @@ void CheckResponse(uint8_t *a, uint8_t *b, uint8_t len_array_cmp)
...
+
+void check_calmode_active() {
+unsigned long currentTime_ms = millis();
+
+  if (digitalRead(BUTTON) == LOW) {
+   if (isLongPress) {
+      if (currentTime_ms > (StartPress_ms + LongPress_ms)) {
+        Serial.println("Start calibration process: 300 seconds of 400 ppm stable");
+        display.clear();
+        display.print("CAL-");
```
Comment (danielbernalb) [inline LibreCO2_SCD30/LibreCO2_SCD30_ArduinoUNO.ino]: The CAL- message only be showed by 100ms, not 5 seconds like before:
delay(5000);

File: LibreCO2_SCD30/LibreCO2_SCD30_ArduinoUNO.ino [hunk 2]
```diff
@@ -158,7 +157,6 @@ void setup()
   display.clear();
   display.print("HEAT");
   Serial.print("Preheat: ");
-  delay (3000);
```
Comment (danielbernalb) [inline LibreCO2_SCD30/LibreCO2_SCD30_ArduinoUNO.ino:161]: The HEAT message not showed, the delay of 3000 seconds was deleted but not replaced

**Review by danielbernalb — CHANGES_REQUESTED** [review]
Two delays deleted but not replaced, messages desapears

**Review by kyogithub — COMMENTED** [review]
Delays distributed

**Pull request merged**